micro-rdk-nmea/src/viamboat/sensors.rs

```diff
@@ -0,0 +1,579 @@
+//! This is a collection of sensors that parse data useful to boats from a sensor providing
+//! raw NMEA message data. The data is expected to be a hashmap with key-value pairs of the form
+//! "<PGN in hex>-<source>": "<raw data encoded as a string in base64>" (example below)
+//!
+//! ```
+//! {
+//!     "1FD0A-16": "Cv0BAHg+gD8voKFnAAAAAIdQAwAAAAAACAD/ABYAAgBbAADgeg8A/w==",
+//!     "1FD0C-23": "DP0BAHg+gD8voKFnAAAAAL+PAwAAAAAACAD/ACMAAgD/AABQkAT//w==",
+//!     "1FD06-16": "Bv0BAHg+gD8voKFnAAAAAFpOAwAAAAAACAD/ABYAAgBb//////YD/w==",
+//!     "1FD06-23": "Bv0BAHg+gD8voKFnAAAAAL1FBAAAAAAACAD/ACMAAgD/1HT//////w==",
+//!     "1FD07-16": "B/0BAHg+gD8voKFnAAAAANNOBAAAAAAACAD/ABYAAgBb/////3/2Aw==",
+//!     "1FD07-23": "B/0BAHg+gD8voKFnAAAAAI9kAwAAAAAACAD/ACMAAgD/wNR0/3///w=="
```

Comment:
This is a very specific framing of the data. What in the codebase is providing it to us like that? How and why did base64 enter the picture?